rpcs3/Emu/RSX/VK/VKGSRender.cpp

```diff
@@ -923,6 +923,9 @@ bool VKGSRender::load_program()
 		}
 	}
 
+	properties.cs.attachmentCount = 1;
```

Comment:
Shouldn't it be m_draw_buffers_count ? I think it's the size of att_state array used during the draw call.